### PR TITLE
Adding BLADE

### PR DESCRIPTION
**Pull request by conraddit**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

**Comment by conraddit** [comment]
Is there something I can do to speed this up?

**Comment by conraddit** [comment]
> > Is there something I can do to speed this up?
> 
> What's the rush? :)

This: https://indigoprotocol1.medium.com/indigo-airdrop-cspa-partnership-1ca79fa3890a

**Review by Bez625 — APPROVED** [review]

File: registry.json [hunk 1]
```diff
@@ -1,9 +1,22 @@
 [{
+ "ticker": "BLADE",
+ "poolId": "d765dee553a7e041d525b1782a10eb0a5e52e96ed55ed173d8bb82a7"
```
Comment (Bez625) [inline registry.json:3]:
```suggestion
 "poolId": "d765dee553a7e041d525b1782a10eb0a5e52e96ed55ed173d8bb82a7",
```

File: registry.json [hunk 2]
```diff
@@ -1,9 +1,22 @@
 [{
+ "ticker": "BLADE",
+ "poolId": "d765dee553a7e041d525b1782a10eb0a5e52e96ed55ed173d8bb82a7",
+ "social": {
+   "twitter": "conraddit"
+     "facebook": "",  
+     "instagram": "",
+     "youtube": ""
+ },
```
Comment (Bez625) [inline registry.json:9]:
```suggestion
     "twitter": "conraddit",
     "facebook": "",  
     "instagram": "",
     "youtube": ""
 },
```

**Pull request merged**